examples/llm-api/ray/simple_ray_single_node.py

```
@@ -0,0 +1,184 @@
+"""
+# for llama
+python simple_ray_single_node.py
+
+# for DeepSeek
+python simple_ray_single_node.py  --model_dir=/scratch/llm-models/DeepSeek-V3-Lite/bf16
+
```

Ruff (0.12.2)

1 blank line required between summary line and description

(D205)